### PR TITLE
Clear un-uploaded changes to Mongo & DocDB documents before opening

### PR DESCRIPTION
**Pull request by wwlorey**
Fixes https://github.com/microsoft/vscode-cosmosdb/issues/1619

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -149,6 +149,8 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
                 node = await ext.tree.showTreeItemPicker<MongoDocumentTreeItem | DocDBDocumentTreeItem>([MongoDocumentTreeItem.contextValue, DocDBDocumentTreeItem.contextValue], actionContext);
             }
 
+            // Clear un-uploaded local changes to the document before opening https://github.com/microsoft/vscode-cosmosdb/issues/1619
+            await node.refresh();
```
Comment (EricJizbaMSFT) [inline src/extension.ts:153]: Our extension isn't caching the local changes, so you don't need to refresh the whole node. Pretty sure VS Code is doing the caching, in which case this should work:
```
ext.fileSystem.fireChangedEvent(node);
```

We call `fireChangedEvent` in `refreshImpl`, which I'm pretty sure is the only reason `node.refresh()` worked in the first place.

File: src/extension.ts [hunk 2]
```diff
@@ -149,6 +149,8 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
                 node = await ext.tree.showTreeItemPicker<MongoDocumentTreeItem | DocDBDocumentTreeItem>([MongoDocumentTreeItem.contextValue, DocDBDocumentTreeItem.contextValue], actionContext);
             }
 
+            // Clear un-uploaded local changes to the document before opening https://github.com/microsoft/vscode-cosmosdb/issues/1619
+            ext.fileSystem.fireChangedEvent(node);
```
Comment (EricJizbaMSFT) [inline src/extension.ts:153]: Actually any chance you could do this in [here](https://github.com/microsoft/vscode-azuretools/blob/master/ui/src/AzExtTreeFileSystem.ts#L40)? If you don't want to do it for this release candidate that's fine, but I think other extensions will have the same problem, so I want it in the shared package eventually

Comment (wwlorey) [reply src/extension.ts:153]: Sounds good. I logged a new issue for it: https://github.com/microsoft/vscode-cosmosdb/issues/1624

Comment (EricJizbaMSFT) [reply src/extension.ts:153]: Okay well this one will take longer to discuss during triage than it'll take to fix. So if you don't want to do it within like a day - I'll just do it myself haha. Let me know

Comment (wwlorey) [reply src/extension.ts:153]: Lol gotcha, I can take care of it right after we ship tomorrow

**Pull request merged**